### DISCUSSION TITLE
Online link in printable meeting list

### DISCUSSION THREAD

**Discussion by liberatus1** [code4recovery/12-step-meeting-list#537] [Ideas]
Hi Team,

can we get the online meeting url added in the printable meeting list? Currently it’s not there.

Thanks!
Jason

**Comment by tech2serve** [comment]
Hey @liberatus1,

I transferred this issue because it doesn’t fit in the `spec` repo.

This does duplicate code4recovery/pdf#2, and we’ll look into this closer.

**Comment by joshreisner** [comment]
Hi @liberatus1 just to confirm, are you talking about the pdf service at pdf.code4recovery.org, or are you talking about some other printable meeting list?

**Comment by liberatus1** [comment]
Hi Josh,

It was in reference to the one here https://www.aainpalmbeach.org/wp-admin/admin-ajax.php?action=tsml_pdf and I didn't know about that one you posted, which looks very useful, is the type I posted now obsolete in favor of the new pdf maker?

I used the pdf maker you posted using our data but I don't see it including zoom links. Also, it looks like the new one doesn't include meeting notes which in some cases are very important?

Our office manager would like to switch our where & when's over to printing dynamically from the online meeting list rather than maintaining 2 separate lists like they have been.

Thanks,
Jason

**Comment by joshreisner** [comment]
ok! moving this to 12-step-meeting-list discussions!

**Comment by joshreisner** [comment]
to provide a little context on this, the PDF feature in the plugin was never really intended for production, it's something I put together as a starting place for a specific central office who had hired a contractor, who to my knowledge never completed their work.

recently, we've created a new service at pdf.code4recovery.org and in the next TSML version we'll be adding a link to it. 

our intention with that though is to create something specifically for in-person meetings, and don't envision adding online meetings to it - [see here for the rationale](https://github.com/code4recovery/pdf/issues/2#issuecomment-927975620), but basically we imagine that people looking for an online meeting will by definition have access to a smartphone or computer.

however this is in the discussions area, so if people have thoughts about it, let's discuss

**Reply by liberatus1** [reply]
Ok, yeah this makes sense to me for sure. I guess our people have been including the zoom info in the where and when, but to me it makes sense to just include a single link in the where and when booklet to the url of the online meeting list so people with a physical copy can see there is an online list, and can use their phone for the rest. Honestly the whole idea of a physical book is not too relevant now much less including online links within it.

Are you planning to add an option to include meeting and/or location notes in the PDF maker? 

Not to get too crazy, I'm not suggesting you do this, but just if in the future based on user requests if it makes sense to make it very flexible per user which is probably not the case but if it was, you could add some simple templating system alongside a list of available data fields like

[item]
%meetingName%
%meetingLocation%
%meetingNotes%
[/item]
[footer]
...
[/footer]

Nice work as always Josh thanks for your amazing service.

Jason
South Palm Beach County Integroup webmaster
